packages/plugins/experimental/plugin-script/src/types/script.ts

```diff
@@ -5,16 +5,23 @@
 import { ref, S, TypedObject } from '@dxos/echo-schema';
 import { TextType } from '@dxos/plugin-markdown/types';
 
+// TODO(burdon): Which is the outer type? I.e., in the navbar?
+
 export class ScriptType extends TypedObject({ typename: 'dxos.org/type/Script', version: '0.1.0' })({
+  // TODO(burdon): Change to URI?
   name: S.optional(S.String),
+  description: S.optional(S.String),
   source: ref(TextType),
+  // TODO(burdon): Change to has of deployed content.
   // Whether source has changed since last deploy.
   changed: S.optional(S.Boolean),
 }) {}
 
 export class FunctionType extends TypedObject({ typename: 'dxos.org/type/Function', version: '0.1.0' })({
+  name: S.optional(S.String),
   version: S.Number,
   // Local binding to a function name.
+  // TODO(burdon): Move binding to inner type.
```

Comment:
Binding should not be on the `ScriptType`, those are only for authoring and not for invoking